catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -796,22 +796,29 @@ QuerySources initializeSources(
           for (String id : sourceIds) {
             LOGGER.debug("Looking up source ID = {}", id);
 
-            final List<FederatedSource> fedSources =
+            List<FederatedSource> sources = new ArrayList<>();
+            sources.addAll(
                 frameworkProperties
                     .getFederatedSources()
                     .stream()
                     .filter(e -> e.getId().equals(id))
-                    .collect(Collectors.toList());
+                    .collect(Collectors.toList()));
+            sources.addAll(
+                frameworkProperties
+                    .getCatalogStores()
```

Comment:
As I said offline, I'd recommend someone more familiar with the Catalog's APIs review this particular change, like @pklinef.

Reply:
I think this should be fine as it is only additive. Just my 2 cents.